Sources/SpeziLocalLLM/LLMError.swift

```diff
@@ -0,0 +1,21 @@
+//
+// This source file is part of the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+/// The ``LLMError`` describes possible errors that occure during the execution of the ``LLM`` via the ``LLMRunner``.
+public enum LLMError: Error {
```

Comment:
I would recommend that the error conforms to [`LocalizedError`](https://developer.apple.com/documentation/foundation/localizederror) to ensure that we can easily use localized descriptions of errors like in other Spezi Packages 👍